### PR TITLE
Solve the exception if the activity implements PermissionAwareActivity

### PR DESCRIPTION
**Pull request by jp928**
## Motivation
To solve the exception if the activity implements PermissionAwareActivity.

### CODE REVIEW

**Comment by roland-mediamonks** [comment]
I still need this permission fix on the most recent version of this module to avoid the crash. Could this be merged?

**Comment by wko27** [comment]
+1

File: android/src/main/java/com/imagepicker/ImagePickerModule.java [hunk 1]
```diff
@@ -585,11 +586,15 @@ else if (activity instanceof OnImagePickerPermissionsCallback)
...
         }
+        else if (activity instanceof PermissionAwareActivity) {
+          ((PermissionAwareActivity) activity).requestPermissions(PERMISSIONS, requestCode, listener);
+        }
         else
         {
           final String errorDescription = new StringBuilder(activity.getClass().getSimpleName())
                   .append(" must implement ")
                   .append(OnImagePickerPermissionsCallback.class.getSimpleName())
+                  .append(PermissionAwareActivity.class.getSimpleName())
```
Comment (haitaoli) [inline android/src/main/java/com/imagepicker/ImagePickerModule.java]: This concatenate two class names together. You can add `" or "` in between.

File: android/build.gradle [hunk 2]
```diff
@@ -51,9 +51,9 @@ dependencies {
...
-    testCompile "org.easytesting:fest-assert-core:${FEST_ASSERT_CORE_VERSION}"
-    testCompile "org.powermock:powermock-api-mockito:${POWERMOCK_VERSION}"
-    testCompile "org.powermock:powermock-module-junit4-rule:${POWERMOCK_VERSION}"
-    testCompile "org.powermock:powermock-classloading-xstream:${POWERMOCK_VERSION}"
-    testCompile "org.mockito:mockito-core:${MOCKITO_CORE_VERSION}"
+    // testCompile "org.easytesting:fest-assert-core:${FEST_ASSERT_CORE_VERSION}"
+    // testCompile "org.powermock:powermock-api-mockito:${POWERMOCK_VERSION}"
+    // testCompile "org.powermock:powermock-module-junit4-rule:${POWERMOCK_VERSION}"
+    // testCompile "org.powermock:powermock-classloading-xstream:${POWERMOCK_VERSION}"
+    // testCompile "org.mockito:mockito-core:${MOCKITO_CORE_VERSION}"
```
Comment (haitaoli) [inline android/build.gradle]: Why are these lines commented out?

**Comment by haitaoli** [comment]
Merged: https://github.com/react-community/react-native-image-picker/pull/848

**Pull request closed without merging**